Swap.jl

```diff
@@ -0,0 +1,66 @@
+include("Solution.jl")
+
+DEBUG = true
+
+struct Swap
+    data::TSP
+    solution::Solution
+ end
+ 
+function nodeCost(route::Vector{Int}, index::Int64, weights::Matrix{Float64})
+    size = length(route)
+    value = route[index]
+
+    if index == 1 
+        prev = route[size]
+    else
+        prev = route[index - 1]
+    end
```

Comment:
Maybe a nitpick, but when defining a variable by a simple if-else clause, I prefer the ternary operator:

```julia 

prev = index == 1 ? route[size] : route[index - 1]

```